fundingmanager.go

```diff
@@ -1071,10 +1079,15 @@ func (f *fundingManager) handleFundingCreated(fmsg *fundingCreatedMsg) {
 		case <-f.quit:
 			// The fundingManager is shutting down, will resume
 			// wait for funding transaction on startup.
-		case <-doneChan:
+		case err := <-errChan:
+			// Encountered an error
+			fndgLog.Errorf("Encountered an error: %v", err)
+		case confDetails := <-confirmChan:
 			// Success, funding transaction was confirmed.
 			f.deleteReservationCtx(peerKey,
 				fmsg.msg.PendingChannelID)
+			// Now we can call processConfirmedFunding
+			f.processConfirmedFunding(completeChan, confDetails)
```

Comment:
this must be done _before_ the reservation context is deleted, to keep the original behavior.